themes/default/subtemplates/entry.inc.tpl

```diff
@@ -24,7 +24,7 @@
 {if (($settings.user_area_access == 0 and ($admin or $mod)) or ($settings.user_area_access == 1 and $user) or $settings.user_area_access == 2) && $posting_user_id>0}
 {assign var=name value="<a href=\"index.php?mode=user&amp;show_user=$posting_user_id\">$name</a>"}
 {/if}
-<article class="posting{if $is_read} read{/if}">
+<article id="p{$id}" class="posting{if $is_read} read{/if}">
```

Comment:
The ID seems unnecessary, as `poTarget` can be defined without the ID (it is unique due to `i`). Moreover, the ID generates invalid HTML code due to a conflict with `<span id="p{$data.$element.id}" class="tail">` (in case of `$data.$element.id == $id`).

Reply:
I didn't realise that there is a conflict. The intention was to add a similar ID like in the thread based posting views. I will solve the conflict.

Reply:
Changed the prefix for the posting ID from `p` to `sp` which stands for *single posting*. Since the ID in the single post view is used exclusively for generating the popovertarget, thus avoiding name collisions with the tail of a posting in the thread tree, this simple change is good enough to fulfil its purpose.